File: Command.java
import java.lang.*;

class Command
{
	public static void main(String Arg[])
	{
		System.out.println("First number is :"+Arg[0]);
		System.out.println("First number is :"+Arg[1]);
		
		int ans=Integer.parseInt(Arg[0])+Integer.parseInt(Arg[1]);
	    System.out.println("Addition is:"+ans);
	}
}

File: Command.c
#include<stdio.h>
#include<stdlib.h>
int main(int argc,char*argv[])
{ int ans=0;
	printf("Name of executable is:%s\n",argv[0]);
	printf("number of command line argument are:%d\n",argc);

	ans=atoi(argv[1])+atoi(argv[2]);
	printf("addition is:%d\n",ans);
	return 0;
}